templates/BlueGrey/compare.tmpl

```diff
@@ -36,7 +36,7 @@
       [websvn-test:newpath]
       <table class="comparison">
         <thead>
-        <tr><th><a href="[websvn:fileurl]">[websvn:newpath]</a></th></tr>
+        <tr><th onclick="collapsefile($(this));"><a href="[websvn:fileurl]">[websvn:newpath]</a></th></tr>
```

Comment:
Remove `onclick` in favour of registering handlers for that event in `collapsible.js` only. That's the far better solution and you already had it! :-) If that doesn't work, let's discuss the reason and try to fix it.

Reply:
Yes i thought by inline you meant something like onclick style function that is there already there and then move the function to separate JS file. :) No issues i have revert this HTML changes and went back to the original solution.